src/tests/functional/plugin/cpu/single_layer_tests/loop.cpp

```diff
@@ -320,10 +320,10 @@ const std::vector<ElementType> inputPrecisions = {
 };
 
 std::vector<InputLayerType> trip_count_type { InputLayerType::CONSTANT, InputLayerType::PARAMETER };
-std::vector<int64_t> trip_count { 1, 5 }; // works only if trip_count_type is constant
+std::vector<int64_t> trip_count { /*0,*/ 1, 5 }; // works only if trip_count_type is constant. Zero count of iteration not supported in reference
```

Comment:
If we can not run a test case due to some reasons we have to add test skip config with the reason about why the test is skipped, usually it is a jira ticket number. Could you please add dedicated test skip config? If we need to extend the reference function with zero trip count support I think we have to create dedicated ticket.
Could you please elaborate how we test `first_exec = false` case? I guess there were no such tests before.

Reply:
Agree. Added tests with `exec_cond = false` and updated `skip_tests_config`. Thanks!